content/python/concepts/functions/terms/isinstance/isinstance.md

```diff
@@ -0,0 +1,61 @@
+---
+Title: 'isinstance()'
+Description: 'Returns `True` if object is the specified type. Otherwise function will return `False`.'
+Subjects:
+  - 'Computer Science'
+  - 'Python'
+  - 'Engineering'
+  - 'Programming'
+  - 'Web Dev'
+  - 'Data Science'
+  - 'Data Analytics'
+Tags:
+  - 'Python'
+  - 'Functions'
+  - 'Built-in'
+  - 'Boolean'
+  - 'Objects'
+  - 'Types'
+  - 'TypeError'
+  - 'Parameters'
+  - 'Class'
+CatalogContent:
+  - 'python'
+  - 'functions/isinstance'
+---
+
+<h1>isinstance()</h1>
```

Comment:
Do not use an h1 header with the title, the metadata provides that. Also do not use HTML. Entries should be written in Markdown.